app-misc/sphinx/sphinx-2.1.9-r1.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WANT_AUTOMAKE=1.15
+
+inherit eutils autotools toolchain-funcs
+
+#MY_P=${P/_/-}
+MY_P=${P}-release
+
+DESCRIPTION="Full-text search engine with support for MySQL and PostgreSQL"
+HOMEPAGE="http://www.sphinxsearch.com/"
+SRC_URI="http://sphinxsearch.com/files/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~ppc-macos ~x86-macos ~sparc-solaris ~sparc64-solaris"
+IUSE="debug id64 mysql odbc postgres stemmer syslog test xml"
+
+RDEPEND="mysql? ( virtual/mysql )
+	postgres? ( dev-db/postgresql:* )
+	odbc? ( dev-db/unixODBC )
+	stemmer? ( dev-libs/snowball-stemmer )
+	xml? ( dev-libs/expat )
+	virtual/libiconv"
+
+S=${WORKDIR}/${MY_P}
+
+src_prepare() {
+	eapply -p0 "${FILESDIR}"/${PN}-2.0.1_beta-darwin8.patch
+
+	# drop nasty hardcoded search path breaking Prefix
+	# We patch configure directly since otherwise we need to run
+	# eautoreconf twice and that causes problems, bug 425380
+	sed -i -e 's/\/usr\/local\//\/someplace\/nonexisting\//g' configure || die
+
+	# Fix QA compilation warnings.
+	sed -i -e '19i#include <string.h>' api/libsphinxclient/test.c || die
+	sed -i -e 's/libsphinxclient.a/libsphinxclient.so/g' api/libsphinxclient/Makefile.am || die
+
+	eapply_user
+
+	pushd api/libsphinxclient || die
+	eautoreconf
+	popd || die
+}
+
+src_configure() {
+	# fix libiconv detection
+	use !elibc_glibc && export ac_cv_search_iconv=-liconv
+
+	econf \
+		--sysconfdir="${EPREFIX}/etc/${PN}" \
+		$(use_enable id64) \
+		$(use_with debug) \
+		$(use_with mysql) \
+		$(use_with odbc unixodbc) \
+		$(use_with postgres pgsql) \
+		$(use_with stemmer libstemmer) \
+		$(use_with syslog syslog) \
+		$(use_with xml libexpat )
+
+	cd api/libsphinxclient || die
+	econf --disable-static STRIP=:
+}
+
+src_compile() {
+	emake AR="$(tc-getAR)" || die "emake failed"
+
+	emake -j 1 -C api/libsphinxclient || die "emake libsphinxclient failed"
+}
+
+src_test() {
+	elog "Tests require access to a live MySQL database and may require configuration."
+	elog "You will find them in /usr/share/${PN}/test and they require dev-lang/php"
+}
+
+src_install() {
+	emake DESTDIR="${D}" install || die "install failed"
+	emake DESTDIR="${D}" -C api/libsphinxclient install || die "install libsphinxclient failed"
+	find "${D}" -name '*.la' -delete || die
+
+	dodoc doc/*
+
+	dodir /var/lib/sphinx
+	dodir /var/log/sphinx
+
+	newinitd "${FILESDIR}"/searchd.rc searchd
+
+	if use test; then
```

Comment:
...why install test stuff?

Reply:
I think it's a question for @graaff

Reply:
@graaff ping

Reply:
@graaff ping

Reply:
sphinx 2.1.9 is old and obsolete, this code has been removed from later versions. This version was only still around to facilitate people moving and because some arches still needed to keyword new dependencies. I have now removed 2.1.9.